Src/Clients/WebUI/wwwroot/css/actors-index.css

```diff
@@ -1,3 +1 @@
-﻿.selected {
-    background-color: cornflowerblue;
-}
\ No newline at end of file
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/manual/liannoi/team-project/pullRequest?prid=5600579)